com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/lib/domain/CloudApplicationExtended.java

```diff
@@ -20,6 +20,7 @@
     private List<ServiceKeyToInject> serviceKeysToInject;
     private List<ApplicationPort> applicationPorts;
     private List<String> domains;
+    private Map<String, Boolean> restartParameters;
```

Comment:
Use a POJO instead of a Map to store restart parameters. That way, people wouldn't have to "remember" where to get the keys from.